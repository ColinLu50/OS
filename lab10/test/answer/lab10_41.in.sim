5140
707
707
448
707
410
