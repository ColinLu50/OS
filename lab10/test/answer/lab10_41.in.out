FCFS
249 -> 215 -> 419 -> 302 -> 66 -> 9 -> 91 -> 446 -> 127 -> 297 -> 146 -> 313 -> 326 -> 352 -> 248 -> 36 -> 115 -> 223 -> 235 -> 458 -> 9 -> 403 -> 398 -> 66 -> 436 -> 395 -> 330 -> 74 -> 0 -> 139 -> 59 -> 325
5140
SSTF
249 -> 248 -> 235 -> 223 -> 215 -> 146 -> 139 -> 127 -> 115 -> 91 -> 74 -> 66 -> 66 -> 59 -> 36 -> 9 -> 9 -> 0 -> 297 -> 302 -> 313 -> 325 -> 326 -> 330 -> 352 -> 395 -> 398 -> 403 -> 419 -> 436 -> 446 -> 458
707
SCAN
249 -> 248 -> 235 -> 223 -> 215 -> 146 -> 139 -> 127 -> 115 -> 91 -> 74 -> 66 -> 59 -> 36 -> 9 -> 0 -> 0 -> 297 -> 302 -> 313 -> 325 -> 326 -> 330 -> 352 -> 395 -> 398 -> 403 -> 419 -> 436 -> 446 -> 458
707
C-SCAN
249 -> 248 -> 235 -> 223 -> 215 -> 146 -> 139 -> 127 -> 115 -> 91 -> 74 -> 66 -> 59 -> 36 -> 9 -> 0 -> 0 -> 496 -> 458 -> 446 -> 436 -> 419 -> 403 -> 398 -> 395 -> 352 -> 330 -> 326 -> 325 -> 313 -> 302 -> 297
448
LOOK
249 -> 248 -> 235 -> 223 -> 215 -> 146 -> 139 -> 127 -> 115 -> 91 -> 74 -> 66 -> 59 -> 36 -> 9 -> 0 -> 297 -> 302 -> 313 -> 325 -> 326 -> 330 -> 352 -> 395 -> 398 -> 403 -> 419 -> 436 -> 446 -> 458
707
C-LOOK
249 -> 248 -> 235 -> 223 -> 215 -> 146 -> 139 -> 127 -> 115 -> 91 -> 74 -> 66 -> 59 -> 36 -> 9 -> 0 -> 458 -> 446 -> 436 -> 419 -> 403 -> 398 -> 395 -> 352 -> 330 -> 326 -> 325 -> 313 -> 302 -> 297
410
